src/sections/StudyResults/styles.scss

```diff
@@ -0,0 +1,109 @@
+@import '../../scss/helpers/variable.scss';
+@import '../../scss/helpers/mixins.scss';
+
+.study-results {
+	color: #fff;
+	&__container {
```

Comment:
it is possible to make one common container class and not copy it from file to file

Reply:
Sure